frontend/apps/reader/modules/readergesture.lua

```diff
@@ -335,7 +337,9 @@ function ReaderGesture:buildMenu(ges, default)
         {"zoom_pageheight", not self.is_docless},
         {"zoom_column", not self.is_docless},
         {"zoom_content", not self.is_docless},
-        {"zoom_page", not self.is_docless},
+        {"zoom_page", not self.is_docless, true},
+
+        {"wallabag_download", self.ui.wallabag ~= nil, true},
```

Comment:
In some next Gesture manager, we should remove that "true" (separator) on the last item, as it's a bit ugly.
The code I added yesterday will add a separator itself at this point, but only when there are custom gestures.

Reply:
The whole thing could probably use some refactoring, but I haven't given it much thought. It was good enough to quickly expand with the basic functionality I wanted without too much trouble.

Reply:
Oups, I mean "in some next gesture manager _PR_" :) No problem with _that_ Gesture manager as it is!